dspace-ldn/src/main/java/org/dspace/app/ldn/processor/LDNMetadataProcessor.java

```diff
@@ -94,6 +96,13 @@ private LDNMetadataProcessor() {
 
     @PostConstruct
     public void init() {
+        String allowedExternalResolverUrls = configurationService.getProperty("ldn.notify.allowed-external-resolver-urls");
```

Comment:
To eliminate the ambiguity, could you name the local String variable, `allowedExternalResolverUrls`, something different than the Array class member, `allowedExternalResolverUrls`?

Reply:
Yeah, I can do that.

Reply:
Done.